worldedit-core/src/main/java/com/sk89q/worldedit/command/GeneralCommands.java

```diff
@@ -271,16 +280,16 @@ public void searchItem(Actor actor,
                                List<String> query) {
         String search = String.join(" ", query);
         if (search.length() <= 2) {
-            actor.printError("Enter a longer search string (len > 2).");
+            actor.printError(TranslatableComponent.of("worldedit.searchitem.too-short"));
             return;
         }
         if (blocksOnly && itemsOnly) {
-            actor.printError("You cannot use both the 'b' and 'i' flags simultaneously.");
+            actor.printError(TranslatableComponent.of("worldedit.searchitem.b-and-i"));
```

Comment:
I feel like the translation key would be better as `.no-block-and-item-flags`, or at least something more verbose.